cloudbuild/README.md

```diff
@@ -30,8 +30,14 @@ To add a dependency for GPU tests:
 - Have a Keras team member update the Docker image for GPU tests by running the remaining steps
 - Create a `Dockerfile` with the following contents:
 ```
-FROM tensorflow/tensorflow:2.9.1-gpu
+FROM tensorflow/tensorflow:2.10.0-gpu
+RUN \
+    apt-get -y update && \
+    apt-get -y install openjdk-8-jdk && \
```

Comment:
out of curiosity -- why is jdk required?

Reply:
This is required for Bazel, which is built in Java.